what-is-all-this.md

```diff
@@ -0,0 +1,23 @@
+The Pyodide runtime consists of the following components, sorted in terms of
```

Comment:
Maybe `repository-structure.md` or something similar?

Reply:
Not a very accurate name yet but maybe if we name it that we can add content to eventually make it a more complete description of the repository structure. I rather like `what-is-all-this.md` but I understand it might not quite be the tone you're aiming for in technical docs.